### PR TITLE
mgr: raise python exception on failure in send_command()

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
it also silences the clang analyzer warning of http://people.redhat.com/bhubbard/scan-build-2017-06-07-193016-13960-1/report-cd0a02.html#EndPath

File: src/mon/OSDMonitor.cc [hunk 1]
```diff
@@ -7090,7 +7090,7 @@ int OSDMonitor::prepare_command_osd_destroy(
   // defying PaxosService and all laws of nature. Therefore, as we may
   // be used during 'osd purge', let's keep the caller responsible for
   // proposing.
-
+  assert(err == 0);
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc]: @badone this silences some clang analyzer warnings.

Comment (badone) [inline src/mon/OSDMonitor.cc]: Nice!

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-06-15_17:39:27-rados-wip-kefu-testing---basic-smithi/ all issues are known ones.

**Pull request merged**